.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = src/ExternalModules/CurlSharp
 	url = https://github.com/Sonarr/CurlSharp.git
 	branch = master
+[submodule "src/ExternalModules/SocksWebProxy"]
+	path = src/ExternalModules/SocksWebProxy
```

Comment:
please avoid submodules, we will be removing other submodules as well, 

is there a reason we can't use https://www.nuget.org/packages/SocksWebProxy/?